docs/source/docformat/restructuredtext/index.rst

```diff
@@ -96,7 +96,7 @@ Here is a list of the supported ReST directives by package of origin:
 
 - `docutils`: ``.. include::``, ``.. contents::``, ``.. image::``, ``.. figure::``, ``.. unicode::``, ``.. raw::``, ``.. math::``, ``.. role::``, ``.. table::``, etc.
 - `epydoc`: None
-- `Sphinx`: None
+- `Sphinx`: ``.. warning::``, ``.. note::`` and other admonitions. 
```

Comment:
I thought that `admonitions` are standard docutils features and not something specific to Sphinx

https://docutils.sourceforge.io/docs/ref/rst/directives.html#specific-admonitions

Reply:
Yeah, but .. note:: etc are actually defined in sphinx only. 
But the concept of the admonition is from docutils, so I guess you're right.